### PR TITLE
added a java code for Queue_Using_Arrays

### LINKED ISSUE #2299: Queue implementation using Array in Java
As a participant of GSSoC, I want to work on this. Please assign this task to me.

### PR DESCRIPTION
**Pull request by eKta37**
Fixes #2299

### CODE REVIEW

**Comment by SakshiShreya** [comment]
Please complete the PR template

**Comment by jainaman224** [comment]
No activity since 2 days.

**Comment by eKta37** [comment]
Fixes #2299 

Checklist:
- 4 space indentation.
- Coding conventions are followed.
- Input is taken dynamically.
- Sample Input / Output is added at the end of file.
- Logic Documentation (Comments).
- File names are correct.

Changes proposed in this pull request:
- Added a code in Java to create a queue using arrays

Languages Used:
- Java

Files Added:
- Queue_Using_Arrays.java
Thanks!

**Comment by somya-kapoor** [comment]
Please use 4 space indentation.

File: Queue_Using_Arrays/Queue_Using_Arrays.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+		}
+		return data[first];
+	}
+		
+	//adds an element in the last
+	public void push(int element) {
+		if(isEmpty()) {
+			first = 0;
+		}
+		if(size ==  data.length) {     //if there is no space left in the queue
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_Using_Arrays.java]: Add comment above the statement.

File: Queue_Using_Arrays/Queue_Using_Arrays.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+	//doubles the size of array/queue
+	private void doubleArray() {
+		int temp[] = data;
+		data = new int[2*data.length];
+		int j = 0;
+		int temp1 = first;
+		for(int i=first; i<temp.length; i++, j++) {
+			data[j] = temp[i];
+		}	
+		for(int i=0; i<=first-1; i++, j++) {
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_Using_Arrays.java]: Please give space around operators

File: Queue_Using_Arrays/Queue_Using_Arrays.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+		return temp;
+	}
+		
+	//doubles the size of array/queue
+	private void doubleArray() {
+		int temp[] = data;
+		data = new int[2*data.length];
+		int j = 0;
+		int temp1 = first;
+		for(int i=first; i<temp.length; i++, j++) {
```
Comment (somya-kapoor) [inline Queue_Using_Arrays/Queue_Using_Arrays.java]: Please give space around operators

**Comment by somya-kapoor** [comment]
Please use 4 space indentation.

**Comment by somya-kapoor** [comment]
@eKta37 Are you working on it?

**Comment by eKta37** [comment]
Yes, I'm working on it.

**Comment by somya-kapoor** [comment]
@eKta37 Please use 4 space indentation, you are using 8 space indentation.

**Comment by somya-kapoor** [comment]
@eKta37 Are you working on it?

**Comment by eKta37** [comment]
@somya-kapoor please review it.

**Comment by somya-kapoor** [comment]
Please use 4 space indentation instead of 8 space.

**Comment by somya-kapoor** [comment]
@eKta37 Are you working on it?

**Comment by somya-kapoor** [comment]
@eKta37 Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**